apps/nextra/pages/en/build/smart-contracts/book/functions.mdx

```diff
@@ -232,8 +232,9 @@ script {
 }
 ```
 
-Entry functions can take primitive types, String, and vector arguments but cannot take Structs (e.g. Option). They also
-must not have any return values.
+Entry functions can accept parameters that are: primitive types, reference to a `signer`, vectors (where the element type is itself acceptable),
+and certain standard library types such as `String`, `Object`, `Option`, `FixedPoint32`, `FixedPoint64`.
```

Comment:
FixedPoint might be supported, but I'm unsure how it's BCS encoded.

Let's skip it for now?

Reply:
Sounds good - I was going based on the code in the extended checker.